### ISSUE TITLE
Error when sending to Sabnzbd

### ISSUE THREAD

**Issue by Yaff1e** [theotherp/nzbhydra2#224]
Since the update to 1.5.2 I'm getting an error when I perform a search and then click the Sabnzbd icon to download. Instead of retrieving the categories I get the following error:-

An error occurred:
500: Internal Server Error

Path: /nzbhydra/internalapi/downloader/SABnzbd/categories

Message: I/O error on GET request for "http://127.0.0.1:8080/api": Failed to connect to /127.0.0.1:8080; nested exception is java.net.ConnectException: Failed to connect to /127.0.0.1:80

It's worked fine on every previous version but not 1.5.2.

Attached is the log file showing the error when I click.

[nzbhydra.log](https://github.com/theotherp/nzbhydra2/files/2257258/nzbhydra.log)

**Comment by theotherp** [comment]
Please post the full debug infos zip

**Comment by Yaff1e** [comment]
Sorry, I completely missed your reply. Attached is the full zip.
[nzbhydra-debuginfos-2018-08-09-16-20.zip](https://github.com/theotherp/nzbhydra2/files/2274769/nzbhydra-debuginfos-2018-08-09-16-20.zip)

**Comment by theotherp** [comment]
Are you sure sabnzbd is configured correctly in Hydra? I can't see its URL and so on because that's sensitive data hidden in the debug infos. It looks like Hydra is trying sabnzbd under http://<somedomainorip>:8080/api. is Sabnzb actually running and reachable under the configured URL and port?

**Comment by Yaff1e** [comment]
Yes, it's reachable. It was set to 127.0.0.1 in Hydra which has been working fine with previous versions, it's only with the current version it's failing. I've just changed it to the servers IP address rather than 127.0.0.1 and it works so I guess something has changed.
